__test__/integration/integration.test.ts

```diff
@@ -103,26 +103,26 @@ describe('integration test', () => {
     it('has the correct content', () => {
       expect(readProp('#link-to-renuo', 'href')).resolves.toEqual('https://www.renuo.ch/');
       expect(readProp('.title', 'innerHTML', 0)).resolves.toEqual('I must review');
-      expect(readProp('.title', 'innerHTML', 1)).resolves.toEqual('Someone must review');
+      expect(readProp('.title', 'innerHTML', 1)).resolves.toEqual('No review requested');
       expect(readProp('.title', 'innerHTML', 2)).resolves.toEqual('All reviews done');
       expect(readProp('.title', 'innerHTML', 3)).resolves.toEqual('Missing Assignee');
-      expect(readProp('.title', 'innerHTML', 4)).resolves.toEqual('Assigned');
+      expect(readProp('.title', 'innerHTML', 4)).resolves.toEqual('Assigned to me');
```

Comment:
Shouldn't "Team must review" also be in this list?

Reply:
Yes 👍